docs/modules/operation/pages/flows/sentinel/message-broker/activemq.adoc

```diff
@@ -26,6 +26,7 @@ config:edit org.opennms.sentinel.controller
 config:property-set location SENTINEL<1>
```

Comment:
This file is missing a top level heading and is not part of the navigation.

Reply:
It is content in a tab (like we use for the different installation OSs), so doesn't get a top level heading and is not part of the nav.

Reply:
ah, missed that.